tests/erica_legacy/samples/grundsteuer_sample_data.py

```diff
@@ -284,25 +285,58 @@ def parse(self):
         return Vertreter.parse_obj(self.build())
 
 
+class SampleBruchteilsgemeinschaft(Builder):
+    name: str
```

Comment:
You could get rid of this (and the extra line in build() below) and just set `self.dict["name"]` directly if you'd like to just to simplify things a little. But it's a purely stylistic choice.

Reply:
This actually fixes my problem in the other comment, so thanks! :D